### PR TITLE
Testing Python 3 support

### PR DESCRIPTION
**Pull request by xr09**
This is mostly the work of 2to3, so maybe there are a few glitches here and there, tests are passing.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import os
-import urlparse
+import urllib.parse
 
-from raygun4py.middleware import flask as flask_raygun
+#from raygun4py.middleware import flask as flask_raygun
```
Comment (abhisuri97) [inline config.py:4]: what was the motivation behind commenting out mentions of raygun4py?

File: manage.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def setup_general():
                 email=Config.ADMIN_EMAIL)
             db.session.add(user)
             db.session.commit()
-            print 'Added administrator {}'.format(user.full_name())
+            print(('Added administrator {}'.format(user.full_name())))
```
Comment (aharelick) [inline manage.py]: looks like there were two sets of parens added here and twice more further down in the file, I think only one set needs to be added?

File: config.py (hunk 1 above)
Comment (xr09) [reply config.py:4]: Sorry, had to comment that in order to try the project, don't have Raygun account.

File: manage.py (hunk 2 above)
Comment (xr09) [reply manage.py]: Yes, 2to3 seems to like it that way, this PR was just a proof of concept no need to merge it, I'm pleased you'll consider Python 3 support for the project.

**Comment by abhisuri97** [comment]
@xr09 if you can fix the issues brought up in comments and fix merge conflicts, i'd be happy to merge 👍

File: config.py (hunk 1 above)
Comment (abhisuri97) [reply config.py:4]: Mind if i re-comment this back in?

File: manage.py [hunk 3]
```diff
@@ -106,7 +106,7 @@ def run_worker():
         password=app.config['RQ_DEFAULT_PASSWORD'])
 
     with Connection(conn):
-        worker = Worker(map(Queue, listen))
+        worker = Worker(list(map(Queue, listen)))
```
Comment (abhisuri97) [inline manage.py:109]: I believe in this case that a list may cause the Worker to fail (I'll take a look at it tonight).

**Review by abhisuri97 — COMMENTED** [review]
we'll get it merged in tonight pending changes.

**Comment by marciorf** [comment]
Hello, 

Is this ready to merge? 
I need python 3 support. Can I help with anything?

**Comment by abhisuri97** [comment]
@marciorf will merge this evening. But yes, mainly ready to merge.

**Comment by abhisuri97** [comment]
Apologies on why this is taking so long. Apparently pip is not installing the sub folders of raygun correctly preventing us from running `pip3 install -r requirements.txt` 

ref: https://github.com/MindscapeHQ/raygun4py/issues/30

**Comment by abhisuri97** [comment]
Ref #106

**Pull request closed without merging**